### PR TITLE
Add front-end admin forms

### LINKED ISSUE #66: [library] twig HTML form for local file system
**What problem are you trying to solve with Submitty**
Create a html twig to copy path from local system into library when managing library.

**Describe the way you'd like to solve this problem**
Create a twig html form to copy path from local system into library folder.

### PR DESCRIPTION
**Pull request by CalebJ2**
Added forms for uploading zip files, git urls, and file paths to the library management page.

This is a work in progress and hasn't been tested with the backend (#96). Creating this pull request so the forms will be available for testing. Fixes #66, #67, #71.

### CODE REVIEW

**Comment by dakriy** [comment]
FYI, the endpoints are ajax endpoints and should be hit with ajax requests. If you guys really want I can change it to use a normal form... but... I feel like ajax is easier...

File: site/app/templates/admin/library/LibraryFilePath.twig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<form class="form-signin" action="{{ filepath_submit_url }}" method="post">
+    <div class="option row">
+        <div class="option-desc col-md-6">
+            <label for="instructions_url">
+                <div class="option-title">File Path</div>
+                <div class="option-alt">What is the file path to the gradeable?</div>
```
Comment (dakriy) [inline site/app/templates/admin/library/LibraryFilePath.twig]: Is this taking a file path to a zip file already on the server? Do I need to make backend support for that?

File: site/app/views/admin/LibraryManagerView.php [hunk 2]
```diff
@@ -2,12 +2,17 @@
```
Comment (dakriy) [inline site/app/views/admin/LibraryManagerView.php]: Oh sweet! You guys already made the view! Should be super easy to hook up and display.

File: site/public/js/drag-and-drop.js [hunk 3]
```diff
@@ -1051,3 +1051,68 @@ function handleUploadCourseMaterials(csrf_token, expand_zip, cmPath, requested_p
         }
     });
 }
+
+function handleUploadGradeablesZip() {
+  var submit_url = buildCourseUrl(['homework/library', 'zip', 'upload']);
```
Comment (dakriy) [inline site/public/js/drag-and-drop.js]: These urls may need to change idk. But that can be done when we tie everything together easy enough.

File: site/app/templates/admin/library/LibraryFilePath.twig (hunk 1 above)
Comment (CalebJ2) [reply site/app/templates/admin/library/LibraryFilePath.twig]: Yeah. I think Course Materials > Upload Course Materials does something like this.

**Pull request merged**